primefaces/src/main/resources/META-INF/resources/primefaces/panel/panel.js

```diff
@@ -124,6 +125,7 @@ PrimeFaces.widget.Panel = PrimeFaces.widget.BaseWidget.extend({
      * Collapses this panel, if not already collapsed.
      */
     collapse: function() {
+        this.header.attr('aria-expanded', false);
```

Comment:
In this file (both changes) you write the ARIA labels manually instead of using the enumeration. This brings me to the question: Why not using the enumeration so that it‘s refactorable and no unwanted writing mistakes can happen?

Reply:
The Enumeration is only available in JAVA code.  in Javascript code we have no such equivalent.

Reply:
And know I proofed that I‘m a java developer without any clue about JS 🙈